README.md

```diff
@@ -1,29 +1,41 @@
 # Bitcoin & Lightning Network `Francophone` Ressources HUB
 
```

Comment:
Why adding `french` content in the same README ?
Should we not create another README for french ?

Reply:
i guess you reviewed only one file , check they are  2 files 
1. README.md default and in writen in french 
2.README-en.md a redirect and written in english

Reply:
checkout the lated commit "Added requested changes"
Thanks @baimamboukar

Reply:
> i guess you reviewed only one file , check they are 2 files
> 
> 1. README.md default and in writen in french
>    2.README-en.md  english version

Ooooohr sorry! My bad